h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1537,9 +1535,10 @@ private int run2(String[] args) throws Exception {
     job.setOutputValueClass(Text.class);
 
     FileInputFormat.addInputPath(job, new Path("ignored"));
```

Comment:
I know this is not part of this change, but why do we need `FileInputFormat.addInputPath(job, new Path("ignored"));` ? Is `ignored` folder used be the job ?

Reply:
It is not used to read anything, this is a hack to make it work. MR requires an input directory.

I am not aware of something like "NullInputFormat" that would be included in the standard Hadoop distributions. We could implement a dummy input format but I don't think it is really worth it since this hack works.

Your thoughts?

Reply:
Oops, I just noticed I misled you. Sorry! We actually already have H2OInputFormat (which is precisely the dummy input format). I will check if we really need the "ignored" folder specification.